### PR TITLE
Update api.py

### PR DESCRIPTION
**Pull request by fredford**

### CODE REVIEW

**Review by timvm1108 — APPROVED** [review]
great stuff!

**Review by timvm1108 — DISMISSED** [review]
still looks great!

**Review by timvm1108 — APPROVED** [review]
Now it looks better, good stuff!

File: manager/backend/api.py [hunk 1]
```diff
@@ -1,177 +1,249 @@
...
+
+		# Create the user
+		user = serializer.save()
+		# Create an authentication token for the provided user
+		token = Token.objects.create(user=user)
+
+		# Create the author object
+		author = Author(
+						token=token,
                         user=user,
```
Comment (kangxi11) [inline manager/backend/api.py:36]: why is this one not indented like the rest?

**Review by kangxi11 — APPROVED** [review]
lgtm except that weird indentation problem

**Pull request merged**